### PR TITLE
docs: Updates for accuracy in Add to Existing Repository page.

### PR DESCRIPTION
**Pull request by anthonyshew**
### Description

There were a few parts to the copy on this page that needed some touching up, according to user feedback.

### Testing Instructions

👀

### CODE REVIEW

File: docs/repo-docs/getting-started/add-to-existing-repository.mdx [hunk 1]
```diff
@@ -179,13 +179,27 @@ Add `.turbo` to your `.gitignore` file. The `turbo` CLI uses these folders for p
...
+### Run tasks with `turbo`
+
+You can now run the tasks you added to `turbo.json` earlier using Turborepo. Using the example tasks from above:
 
 ```bash title="Terminal"
 turbo build check-types
 ```
 
-This runs `build` and `check-types` at the same time.
+This runs the `build` and `check-types` tasks at the same time, respecting the dependency graph of your project.
```
Comment (chris-olszewski) [inline docs/repo-docs/getting-started/add-to-existing-repository.mdx:202]: I feel like "respecting the dep graph of your project" is a little vague. Does focusing on "shared tasks" feel better to you? 

```suggestion
This runs the `build` and `check-types` tasks at the same time, reusing results from any shared tasks.
```

File: docs/repo-docs/getting-started/add-to-existing-repository.mdx [hunk 2]
```diff
@@ -179,13 +179,27 @@ Add `.turbo` to your `.gitignore` file. The `turbo` CLI uses these folders for p
 
 </Step>
 <Step>
-### Run the `build` and `check-types` tasks with `turbo`
+### Add a `packageManager` field to root `package.json`
+
+Turborepo optimizes your repository using information from your package manager. To declare which package manager you're using, add a [`packageManager`](https://nodejs.org/api/packages.html#packagemanager) field to your root `package.json` if you don't have one already.
```
Comment (chris-olszewski) [inline docs/repo-docs/getting-started/add-to-existing-repository.mdx:184]: Should we advertise [--dangerously-disable-package-manager-check](https://turbo.build/repo/docs/reference/run#--dangerously-disable-package-manager-check) here? I don't want to encourage people to use it, but if for some reason this is a blocker for people we should try to help them unblock themselves.

**Review by chris-olszewski — APPROVED** [review]
LGTM! Just 2 questions, but can ship as is.

File: docs/repo-docs/getting-started/add-to-existing-repository.mdx (hunk 1 above)
Comment (anthonyshew) [reply docs/repo-docs/getting-started/add-to-existing-repository.mdx]: Yes.

File: docs/repo-docs/getting-started/add-to-existing-repository.mdx (hunk 2 above)
Comment (anthonyshew) [reply docs/repo-docs/getting-started/add-to-existing-repository.mdx:184]: Yes.

**Pull request merged**